Source/Scene/Scene.js

```diff
@@ -2562,6 +2562,9 @@ define([
 
         us.update(frameState);
 
+        var backgroundColor = defaultValue(scene.backgroundColor, Color.BLACK);
+        us.updateBackgroundColor(backgroundColor);
```

Comment:
Yes, it seemed that way, but the line below that which I changed was doing the same check - possible for scene.backgroundColor to become undefined, or just paranoia?

-        updateAndExecuteCommands(scene, passState, defaultValue(scene.backgroundColor, Color.BLACK));